### PR TITLE
Issue #13321: Kill mutation for pitest-utils profile

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13321: Kill mutation for pitest-utils profile

### CODE REVIEW

**Comment by romani** [comment]
Adding all in not good option, as it will make execution to be hours. Please keep adding checks tests one by one to keep execution time minimized.

**Comment by Kevin222004** [comment]
Yes this is just for testing purpose i am not going to add all

File: src/test/resources/com/puppycrawl/tools/checkstyle/utils/checkutil/InputCheckUtil3.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        // nesting == 1
+        if (true) {
+            if (true) {
+            }
+        }
+
+        // nesting == 2
+        if (true) {
+            if (true) {
+                if (true) { // violation
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/utils/checkutil/InputCheckUtil3.java:26]: Please use message for each violation

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/utils/checkutil/InputCheckUtil3.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/resources/com/puppycrawl/tools/checkstyle/utils/checkutil/InputCheckUtil3.java]: Done

**Comment by Kevin222004** [comment]
Please re-run failed ci
semaphore is also not looking related to changes

**Comment by Kevin222004** [comment]
Failure of Semaphore 

```
[INFO] Across all seeds:18:04
[INFO] com.puppycrawl.tools.checkstyle.checks.whitespace.SingleSpaceSeparatorCheckTest#testNoStackoverflowError18:04
[INFO] Test results can be found at: 18:04
[INFO] file:///home/semaphore/checkstyle/.nondex/mqPUyJg3YLld6jAMyRVk7sXkJs4gRpxIwGKZagQ7HkI=/test_results.html18:04
[INFO] [NonDex] The id of this run is: mqPUyJg3YLld6jAMyRVk7sXkJs4gRpxIwGKZagQ7HkI=18:04
[INFO] ------------------------------------------------------------------------18:04
[INFO] BUILD FAILURE18:04
[INFO] ------------------------------------------------------------------------18:04
[INFO] Total time:  18:02 min18:04
[INFO] Finished at: 2023-07-21T07:11:03Z18:04
[INFO] ------------------------------------------------------------------------
```

I have faced `[INFO] com.puppycrawl.tools.checkstyle.checks.whitespace.SingleSpaceSeparatorCheckTest#testNoStackoverflowError18:04` in my local many time as well. while generating reports

**Review by romani — APPROVED** [review]
ok to merge.

**Pull request merged**